discord-roles.md

```diff
@@ -0,0 +1,71 @@
+| Steering Council       | SC Members                        | Steering Council role, as we currently have it.                   | :white_check_mark: |        None         |                None                |
+| Moderators             | Moderators                        | Has moderation permissions, i.e. mute, kick, ban, etc.            | :white_check_mark: |        None         |                None                |
+| Maintainers            | Maintainers                       | Has GitHub permissions.                                           | :white_check_mark: |        None         |                None                |
+| Foundation Team        | Foundation Members                | Involved in the NeoForged legal entity.                           | :white_check_mark: |        None         |                None                |
+| Infrastructure Team    | Infrastructure Managers           | Involved in the project infrastructure in some way.               | :white_check_mark: |        None         |                None                |
+| Docs Team              | Documentation writers             | People with permissions on the docs repo.                         | :white_check_mark: |        None         |                None                |
+| Bot Team               | Bot managers                      | People who can manage the bot.                                    | :white_check_mark: |        None         |                None                |
+| NeoForged Team         | All team members                  | Catch-all for all the hoisted roles above.                        |        :x:         |        None         |                None                |
+| Automation             | Camelot                           | Contains camelot so that it can moderate the public.              |        :x:         |        Gray         |                Gear                |
+| Mojang Team            | Mojangsters                       | Mojang people.                                                    |        :x:         |     Mojang Red      |                None                |
+| ModLauncher Team       | McModLauncher Team                | Members of the ModLauncher side project.                          |        :x:         |        None         |                None                |
+| Moderator              | Moderators                        | The pingable moderator role.                                      |        :x:         |        None         |                None                |
+| Probation              | New team members                  | Given to new team members.                                        |        :x:         |        None         |                None                |
+| Helper                 | Experts in the community          | Given to experts.                                                 |        :x:         |        None         |                None                |
```

Comment:
As I've stated in internal discussions, we can rename this role to be something else than "Helper" if we award this distinction to experts and knowledgable members of our community, which do not carry the expectation of needing to help out in the support channels (or be pinged for support).

Reply:
I have removed this role for now, until we come up with a better solution